src/main/java/net/pms/util/SubtitleUtils.java

```diff
@@ -622,4 +622,28 @@ public static String convertColourToASSColourString(int colour) {
 		outputString.append("&H").append(colourString.substring(6, 8)).append(colourString.substring(4, 6)).append(colourString.substring(2, 4));
 		return outputString.toString();
 	}
+
+	/**
+	 * Remove tags like <"b"> <"/b"> <"i"> <"/i"> <"u"> <"/u"> <"/font> <"font color="....">
+	 * from subtitles file for renderer not capable to show that tags correctly.
+	 *
+	 * @param file the source subtitles
+	 * @return Temporary file with the converted subtitles. This file will be deleted when the UMS will be stopped.
+	 */
+	public static File removeTagsFromSubs(File file) throws IOException {
+		BufferedReader input = FileUtil.bufferedReaderWithCorrectCharset(file);
+		File tempSubs = new File(configuration.getTempFolder(), file.getName());
+		BufferedWriter output = new BufferedWriter(new OutputStreamWriter(new FileOutputStream(tempSubs), Charset.forName(CHARSET_UTF_8)));
+		String line;
+		while ((line = input.readLine()) != null) {
+			line = line.replaceAll("\\<.*?>","") + "\n";
```

Comment:
This will affect text content containing ```< >```. I can't find a proper format description so I don't know if there is an escape mechanism, but since the allowed tags are very few I'd think it would be better to only match the specific tags.

From [Wikipedia](https://en.wikipedia.org/wiki/SubRip#SubRip_text_file_format):
```
Formatting

Unofficially the format has very basic text formatting, which can be either interpreted or passed through for rendering depending on the processing application. Formatting is derived from HTML tags for bold, italic, underline and color:[10][dubious – discuss]

    Bold – <b> ... </b> or {b} ... {/b}
    Italic – <i> ... </i> or {i} ... {/i}
    Underline – <u> ... </u> or {u} ... {/u}
    Font color – <font color="color name or #code"> ... </font> (as in HTML)

Nested tags are allowed; some implementations prefer whole-line formatting only.
```